feconf.py

```diff
@@ -235,6 +237,11 @@ def get_empty_ratings():
     },
 }
 
+# Prefixes for activity ids to denote the type of activity. These values should
+# not be changed.
+ACTIVITY_ID_PREFIX_EXPLORATION = 'e:'
```

Comment:
Why not just store the activity types in the backend model instead of this? Doing string logic to determine the type seems messy to me.